.NET/Microsoft.Recognizers.Text.DateTime/Constants.cs

```diff
@@ -57,6 +57,9 @@ public static class Constants
         public const string MID_MOD = "mid";
         public const string LATE_MOD = "end";
 
+        public const string MORE_THAN_MOD = "more than";

```

Comment:
Wouldn't "more" and "less" be better?

Reply:
Can you also add a code comment describing in which situation they would be used?